packages/msi/oh-my-posh.wxs

```diff
@@ -0,0 +1,39 @@
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
+	<Package Language="1033" Manufacturer="Jan De Dobbeleer" Name="Oh My Posh" UpgradeCode="b2a6bcaa-bda3-4c7f-8dec-d8665d7a9b69" Version="$(env.VERSION)" UpgradeStrategy="none">
+		<SummaryInformation Description="https://ohmyposh.dev" />
+		<MediaTemplate CabinetTemplate="tmpl{0}.cab" EmbedCab="yes" />
+		<SetProperty Id="ProgramFiles64Folder" Value="[LocalAppDataFolder]" Sequence="first" Before="CostFinalize" Condition="MSIINSTALLPERUSER=1" />
```

Comment:
You shouldn't need this. Windows Installer will virtualize the directory based on the context: <https://learn.microsoft.com/windows/win32/msi/installation-context#folder-redirection>

Reply:
I might need this if the actual installation path deviate from Inno, otherwise there are two installations. Validating this later today.